src/pages/SearchPage/index.js

```diff
@@ -28,7 +28,7 @@ const SearchPage = () => {
         className="show-link"
         key={i}
         onClick={() => {
-          navigate(`/search/${s.full_name}`);
+          navigate(`/${s.full_name}`);
```

Comment:
Oh i see you removed it here, for some reason it doesn't show up in params (unless that is your intention)?